docs/workforce-management-framework/topics/pro-running-the-demo-app.adoc

```diff
@@ -0,0 +1,80 @@
+[id='pro-running-the-demo-app-{chapter}']
+= Introduction
+
+This section includes:
+
+. xref:pro-cloning-the-git-repository-{chapter}[Cloning the Git Repository]
+. xref:pro-installing-angular-dependencies-{chapter}[Installing Angular Dependencies]
+. xref:pro-installing-core-dependencies-{chapter}[Installing Core Dependencies]
+. xref:pro-starting-the-core-server-{chapter}[Starting the Core Server]
+. xref:pro-running-the-mobile-and-portal-demo-applications-{chapter}[Running the Mobile and Portal Demo Applications]
+
+= Prerequisites
+
+* Node and npm installed (tested on Node {WFM-RC-NodeVersion})
+* git installed
+* access to `git@github.com:feedhenry-raincatcher/raincatcher-angularjs.git`
+* MongoDb installed
+* Redis installed
+
+= Procedure
+
+[id='pro-cloning-the-git-repository-{chapter}']
+[discrete]
+== Cloning the Git Repository
+
+[source,bash]
+----
+git clone git@github.com:feedhenry-raincatcher/raincatcher-angularjs.git
+
+cd raincatcher-angularjs
+----
+
+[id='pro-installing-angular-dependencies-{chapter}']
+[discrete]
+== Installing Repository Dependencies
+
+[source,bash]
+----
+npm install
+npm run bootstrap
+----
+
+[id='pro-installing-core-dependencies-{chapter}']
+[discrete]
+== Installing Core Dependencies
+
+[source,bash]
+----
+cd ./core
+npm install
+npm run bootstrap
+----
+
+NOTE: The Core repository is automatically cloned by the git command `npm run bootstrap`.
+To work with a different branch of the Core repository, checkout in another branch.
+
```

Comment:
Change 
````
NOTE: The Core repository is automatically cloned by the git command `npm run bootstrap`.
 +To work with a different branch of the Core repository, checkout in another branch.
````
To
````
NOTE: The Core repository is automatically cloned by the command `npm run bootstrap`.
````
it is not a git command and the second sentence doesn't make sense